common/src/main/scala/models/Registration.scala

```diff
@@ -1,9 +1,23 @@
 package models
 
-case class Registration(deviceId: String, platform: Platform, userId: UniqueDeviceIdentifier, topics: Set[Topic])
+case class Registration(deviceId: String, platform: Platform, udid: UniqueDeviceIdentifier, topics: Set[Topic])
 
 object Registration {
   import play.api.libs.json._
 
-  implicit val jf = Json.format[Registration]
+  implicit val writes = Json.writes[Registration]
+
+  implicit val reads = new Reads[Registration] {
+    override def reads(json: JsValue): JsResult[Registration] =
+      replaceUserId(json).flatMap(basicReader.reads)
+
+    private val basicReader = Json.reads[Registration]
+
+    private def replaceUserId(json: JsValue) = json.validate[JsObject] map { o =>
```

Comment:
I was trying to convince myself that a deprecation warning would be more appropriate, but a comment is probably best.